spec/requests/users_spec.rb

```diff
@@ -6,32 +6,31 @@
       get '/users'
     end
     it 'returns http success' do
-      get '/users'
       expect(response).to have_http_status(:success)
     end
     it 'renders the correct template' do
       expect(response).to render_template('index')
     end
-    it 'has the correct text placeholderss' do
+    it 'has the correct text placeholders' do
       body = response.body
-      expect(body).to include('Users#index')
+      expect(body).to include('Users')
     end
   end
 
   describe 'GET /show' do
     before :each do
-      get '/users/1'
+      @user = User.create(name: 'username test', photo: 'photo/test', bio: 'this is index test', postsCounter: 4)
+      get user_path(@user)
     end
     it 'returns http success' do
-      get '/users/1'
       expect(response).to have_http_status(:success)
     end
     it 'renders the correct template' do
       expect(response).to render_template('show')
     end
     it 'has the correct text placeholderss' do
       body = response.body
-      expect(body).to include('Users#show')
+      expect(body).to include(@user.name)
     end
   end
 end
```

Comment:
- [ ] Let's once more together kindly revisit the requirements of this project. Based on the requirements of this project, we are supposed to write featured tests for the `User show page` that affirms the following:

- See the user's profile picture.
- See the user's username.
- See the number of posts the user has written.
- See the user's bio.
- See the user's first 3 posts.
- See a button that lets me view all of a user's posts.
- Tests that redirect to the post's show page when you click on a user's post.
- Tests that redirect to the user's post's index page when you click on a **see all posts**.

Reply:
same thing here you did not even look at my work.